openedx/features/job_board/forms.py

```diff
@@ -16,3 +16,13 @@ class Meta:
             'compensation': RadioSelect,
             'hours': RadioSelect,
         }
+        labels = {
+            'city': 'City:',
+            'country': 'Country:',
+            'application_link': 'APPLICATION LINK:',
+            'website_link': 'WEBSITE LINK:',
```

Comment:
@zahid-arbisoft please make sure that label CASES are also tackled in css.

Reply:
We are using `field` template for all fields, if you want CSS property as well, then I will have to manually add these two fields, their label, error message etc.

Reply:
We have written css to style these labels, please [have a look](https://github.com/philanthropy-u/philu-edx-theme/blob/fe9b3b5205c373665679cae5bfab9669b69fbec7/edx-platform/philu/lms/static/sass/features/job-board/_job-board.scss#L954).

Reply:
Its done in this [commit](https://github.com/philanthropy-u/philu-edx-theme/pull/1367/commits/b520e92894682233a70055ad565d4f3452d432ce) in theme